src/CheckboxGroup/CheckboxGroup.features.stories.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react'
+import {Checkbox, CheckboxGroup, FormControl, Box} from '..'
+
+export default {
+  title: 'Components/CheckboxGroup/Features',
```

Comment:
Do we want to include `component` and `subcomponent` keys on the feature/example stories or nah?